api/EXT_texture_compression_rgtc.json

```diff
@@ -14,7 +14,7 @@
             "version_added": false
           },
           "firefox": {
-            "version_added": "65"
+            "version_added": false
```

Comment:
I'd like to see more evidence of "65" being wrong.

Reply:
Looks like this is only implemented on macOS.

Reply:
Aha, I guess we go with partial_implementation then?